### PR TITLE
Move tests to use MovementBaseSpec on integration tests.

### PR DESCRIPTION
**Pull request by bambuchaAdm**
Move unit controller tests to unit package

### CODE REVIEW

File: test/controllers/consolidations/AssociateDucrSummaryControllerSpec.scala [hunk 1]
```diff
@@ -26,52 +25,29 @@ import forms.{AssociateDucr, Choice, MucrOptions}
...
-  private val messagesApi: MessagesApi = app.injector.instanceOf[MessagesApi]
-  private implicit val messages: Messages = messagesApi.preferred(FakeRequest())
-
   private val routingHelper = RoutingHelper(app, uri)
 
   private trait Test {
     implicit val headerCarrierMock = mock[HeaderCarrier]
-    reset(authConnectorMock, submissionServiceMock, customsCacheServiceMock)
-    when(customsCacheServiceMock.fetchAndGetEntry[Choice](any(), meq(Choice.choiceId))(any(), any(), any()))
+    reset(authConnectorMock, mockSubmissionService, mockCustomsCacheService)
```
Comment (MaciejRewera) [inline test/controllers/consolidations/AssociateDucrSummaryControllerSpec.scala]: Why this change? Now it seems that 'mock' is a verb instead of noun.

Comment (bambuchaAdm) [reply test/controllers/consolidations/AssociateDucrSummaryControllerSpec.scala]: It was just local inline from MovementBaseSpec
